### DISCUSSION TITLE
New Install in Pi

### DISCUSSION THREAD

**Discussion by jjm888** [robske110/IDDataLogger#47] [Q&A] [answered]
Just completed fresh install in a new Raspberry Pi. All seems good with connection to VW and running on Pi. When I go to Web page on local network it opens login Username Password which I setup but when I click login nothing happens... Any advice? Thanks

**Comment by JimKnopf1503** [comment, marked as answer]
Hi!
Check the php version on your Pi. (php -v)
You need to have installed php 8.x and it must be used! If you have php 7 installed parallel to php 8 it may be activ.
You should create a file named phpinfo.php in your webspace (/var/www/html).
Content:
```
<?php
phpinfo();
```

Then open this page in your browser http://yourIP/phpinfo.php
There you can see, which php version is activ for your webpages.
Edit:
If it is the wrong version, you can deaktivate it with:
sudo a2dismod php7.4 // where 7.4 should be replaced with your active version, which you found out using phpinfo.php
Enable the right one with:
sudo a2enmod php8.0 // 8.0 should also replaced with your installed version
sudo systemctl restart apache2
And don't forget to delete the phpinfo.php file afterwards:
sudo rm /var/www/html/phpinfo.php

If you don't have PHP8.x at all, you can find a german howto under:
[https://www.howtoforge.de/anleitung/wie-installiere-ich-php-8-unter-debian-11/](url)

Burkhard

**Reply by jjm888** [reply]
Thanks a bit of work around installing and uninstall but got there

**Reply by jjm888** [reply]
As an update still not working with login...
Updated to php8.1 could log in but IDDatalogger would no longer run on the Pi. Fresh install of everything php7.4 App runs on Pi but no login on Web page. Removed php7.4 and installed php8.0 App works on Pi but still no login...

**Reply by JimKnopf1503** [reply]
Hi!
Next step would be to check, if the database works together with php8.x.
sudo apt install php8.0-pgsql // the number should be equal to your php version
This may solve the problem.
On your phpinfo.php page, far down, there is the headline **PDO**. Below there ist the line PDO drivers. pgsql should be listed there.

**Reply by jjm888** [reply]
That's it. Done fresh install. And added the extra. Grateful
